packages/types/src/index.ts

```diff
@@ -93,6 +93,8 @@ export type TransitVehicle = {
   routeShortName?: string;
   routeLongName?: string;
   routeType?: string;
+  routeColor?: string;
+  highlightColor?: string;
```

Comment:
Sort props.

Reply:
Very happy to sort these. Should I sort all of the props for TransitVehicle, or do it some other way? It seems like they're currently somewhat logically grouped, rather than alphabetically. Even `routeShortName`/`'routeLongName`/`routeType` (right above the two I added) are not alphabetical.

Reply:
No need to worry about the code that was not touched!

Reply:
Sounds great - updated!